src/services/auth/service.ts

```diff
@@ -59,10 +60,19 @@ export class AuthService {
 
     const translated = this.mailer.translate(lang);
     const subject = translated(MAIL.SIGN_UP_TITLE);
+    const greetingsAndSignupText = `${translated(MAIL.GREETINGS)} ${translated(MAIL.SIGN_UP_TEXT)}`;
     const html = `
-    ${this.mailer.buildText(translated(MAIL.GREETINGS))}
-    ${this.mailer.buildText(translated(MAIL.SIGN_UP_TEXT))}
+    ${this.mailer.buildText(greetingsAndSignupText)}
     ${this.mailer.buildButton(link, translated(MAIL.SIGN_UP_BUTTON_TEXT))}
+    ${this.mailer.buildText(
+      translated(MAIL.USER_AGREEMENTS_MAIL_TEXT, {
+        SIGN_UP_BUTTON_TEXT: translated(MAIL.SIGN_UP_BUTTON_TEXT),
+        GRAASP_LANDING_PAGE_ORIGIN: GRAASP_LANDING_PAGE_ORIGIN,
+      }),
+      // Add margin top of -15px to remove 15px margin bottom of the button.
+      // TODO: If the margin bottom of the button can be removed in "layout.ts", remove the style here.
```

Comment:
Did you check the todo? But I guess it's okay to keep -15px, it's not too dramatic.

Reply:
No, I forgot to discuss with you about that. I also think it is okay, so I will remove the TODO ;).